queries/julia/highlights.scm

```diff
@@ -0,0 +1,131 @@
+(identifier) @variable
+
+[
+  (triple_string)
+  (string)
+] @string
+
+(string
+  prefix: (identifier) @constant.builtin)
+
+
+
+(macro_identifier) @function.macro
+(macro_definition 
+  name: (identifier) @function.macro
+  ["macro" "end" @keyword])
+
+(function_definition
+   name: (identifier) @function)
+(call_expression
+   (identifier) @function)
+(call_expression
+   (field_expression (identifier) @method))
+(parameter_list
+    (identifier) @parameter)
+(typed_parameter
+    (identifier) @parameter
+    (identifier) @type)
+(argument_list
+ (typed_expression
+  (identifier) @parameter
+  (identifier) @type))
+
+(type_argument_list
+ (identifier) @type)
+((identifier) @type
+ (match? @type "^[A-Z]"))
+(typed_expression
+ (identifier) @variable
+ (identifier) @type)
+
+
+(field_expression
+  (identifier)
+  (identifier) @field)
+
+(number) @number
+(coefficient_expression
+  (number)
+  (identifier) @constant.builtin)
+
+[
+    ;(_power_operator)
+    ;(_times_operator)
+    ;(_plus_operator)
+    ;(_arrow_operator)
+    ;(_comparison_operator)
+    ;(_assign_operator)
```

Comment:
this does not work, since they are not strictly supertypes (they would be if grammar.js would drop the token in front of choice).